steps/set-macos-timezone/0.0.1/step.yml

```diff
@@ -0,0 +1,28 @@
+title: Set macOS Time Zone
+summary: Set macOS Time Zone
+description: Set macOS Time Zone
+website: https://github.com/chrisfuller/bitrise-step-set-macos-timezone
+source_code_url: https://github.com/chrisfuller/bitrise-step-set-macos-timezone
+support_url: https://github.com/chrisfuller/bitrise-step-set-macos-timezone/issues
+published_at: 2019-08-24T17:17:32.034927-07:00
+source:
+  git: https://github.com/chrisfuller/bitrise-step-set-macos-timezone.git
+  commit: 916a99e41ef22fc35a9444c9578732f08be609f2
+host_os_tags:
+- osx
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
```

Comment:
It seems that shebang in entry file should specify `bash` instead of zsh.

Reply:
Fixed in `v0.0.2` … #2101